NAMESPACE

```diff
@@ -42,11 +41,9 @@ export(timer_available_cancers)
 export(xCell.data)
 import(ComICS)
 import(dplyr)
-import(limSolve)
 import(mMCPcounter)
 import(magrittr)
 import(methods)
-import(preprocessCore)
```

Comment:
I have no idea why preprocessCore and limSolve have been removed from the NAMESPACE, i just documented again...